### PR TITLE
Fix Generate Adhoc SQL Report action output

### PR DESCRIPTION
**Pull request by tschmidtke-r7**
## Proposed Changes
Fixes the `file` type to include the `content` key

### Description
The `file` type in the plugin spec was missing a `content` key making it unable to be referenced from within the UI. Also fixes some linting issues

Describe the proposed changes:

  - Fix the `file` type

### Testing
n/a

Any testing information goes here.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style
n/a

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

**Comment by tschmidtke-r7** [comment]
make validate:

<details validation>
```
(v3) [(ruby-2.6.3)] ~/projects/insightconnect-plugins/rapid7_insightvm <fix/insightvm> ✗ make validate
[*] Use ``make menu`` for available targets
[*] Including available Makefiles: ../tools/Makefiles/Colors.mk ../tools/Makefiles/Helpers.mk
--
[*] Running validators
[*] Validating plugin at .

[*] Running Integration Validators...
[*] Executing validator HelpValidator
[*] Executing validator ChangelogValidator
[*] Executing validator RequiredKeysValidator
[*] Executing validator UseCaseValidator
[*] Executing validator SpecPropertiesValidator
[*] Executing validator SpecVersionValidator
[*] Executing validator FilesValidator
[*] Executing validator TagValidator
[*] Executing validator DescriptionValidator
[*] Executing validator TitleValidator
[*] Executing validator VendorValidator
[*] Executing validator DefaultValueValidator
[*] Executing validator IconValidator
[*] Executing validator RequiredValidator
[*] Executing validator VersionValidator
[*] Executing validator DockerfileParentValidator
[*] Executing validator LoggingValidator
[*] Executing validator ProfanityValidator
[*] Executing validator AcronymValidator
[*] Executing validator JSONValidator
[*] Executing validator OutputValidator
[*] Executing validator RegenerationValidator
[*] Executing validator HelpInputOutputValidator
[*] Executing validator SupportValidator
[*] Plugin successfully validated!

----
[*] Total time elapsed: 2080.342ms

[*] Validating spec with js-yaml
[SUCCESS] Passes js-yaml spec check


[*] Validating markdown...
[SUCCESS] Passes markdown linting

[*] Validating python files for style...
[SUCCESS] Passes flake8 linting

[*] Validating python files for security vulnerabilities...
[main]	INFO	profile include tests: None
[main]	INFO	profile exclude tests: None
[main]	INFO	cli include tests: None
[main]	INFO	cli exclude tests: None
[main]	INFO	running on Python 3.7.6
264 [0.. 50.. 100.. 150.. 200.. 250.. ]
Run started:2020-03-11 19:45:56.025645

Test results:
>> Issue: [B105:hardcoded_password_string] Possible hardcoded password: 'shared_secret'
   Severity: Low   Confidence: Medium
   Location: ./komand_rapid7_insightvm/actions/generate_shared_secret/schema.py:15
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b105_hardcoded_password_string.html
14	class Output:
15	    SHARED_SECRET = "shared_secret"
16

--------------------------------------------------
>> Issue: [B608:hardcoded_sql_expressions] Possible SQL injection vector through string-based query construction.
   Severity: Medium   Confidence: Low
   Location: ./komand_rapid7_insightvm/actions/top_remediations/action.py:134
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b608_hardcoded_sql_expressions.html
133	    def remediations_query(limit):
134	        return f"SELECT fr.solution_id, fr.assets, fr.vulnerabilities, fr.riskscore, ds.nexpose_id, " \
135	               f"ds.summary AS \"summary\", htmlToText(ds.fix) AS \"fix\", " \

--------------------------------------------------
>> Issue: [B608:hardcoded_sql_expressions] Possible SQL injection vector through string-based query construction.
   Severity: Medium   Confidence: Low
   Location: ./komand_rapid7_insightvm/actions/top_remediations/action.py:147
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b608_hardcoded_sql_expressions.html
146	    def assets_query(limit):
147	        return f"WITH criticality_tags AS ( " \
148	               f"SELECT asset_id, array_to_string(array_agg(distinct tag_name),',') AS criticality_tag " \

--------------------------------------------------
>> Issue: [B608:hardcoded_sql_expressions] Possible SQL injection vector through string-based query construction.
   Severity: Medium   Confidence: Low
   Location: ./komand_rapid7_insightvm/actions/top_remediations/action.py:169
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b608_hardcoded_sql_expressions.html
168	    def vulnerabilities_query(limit):
169	        return f"WITH remediation_vulnerabilities AS ( " \
170	               f"SELECT DISTINCT solution_id, vulnerability_id " \

--------------------------------------------------
>> Issue: [B608:hardcoded_sql_expressions] Possible SQL injection vector through string-based query construction.
   Severity: Medium   Confidence: Low
   Location: ./komand_rapid7_insightvm/triggers/new_scans/trigger.py:104
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b608_hardcoded_sql_expressions.html
103	    def scans_query(scan_statuses, site_ids):
104	        return f"SELECT ds.scan_id, dss.description as status, dsscan.site_id, dsite.name as site_name " \
105	               f"FROM dim_scan AS ds " \

--------------------------------------------------

Code scanned:
	Total lines of code: 25068
	Total lines skipped (#nosec): 0

Run metrics:
	Total issues (by severity):
		Undefined: 0.0
		Low: 1.0
		Medium: 4.0
		High: 0.0
	Total issues (by confidence):
		Undefined: 0.0
		Low: 4.0
		Medium: 1.0
		High: 0.0
Files skipped (0):
[FAIL] Fails bandit security checks

[*] Checking for typos with Misspell...
[SUCCESS] Passes Misspell check
```
</details>

File: rapid7_insightvm/plugin.spec.yaml [hunk 1]
```diff
@@ -1653,7 +1658,7 @@ actions:
     output:
       id:
         description: Scan ID
-        title: Identifier of the resource created
+        title: Identifier of the Resource Created
```
Comment (lmilby-r7) [inline rapid7_insightvm/plugin.spec.yaml]:
```suggestion
        title: Scan ID
```
flip the description and title

File: rapid7_insightvm/plugin.spec.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_insightvm
 title: Rapid7 InsightVM
 description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin can get top remediations, scan results and start scans
-version: 3.6.0
+version: 3.6.1
```
Comment (lmilby-r7) [inline rapid7_insightvm/plugin.spec.yaml]:
```suggestion
version: 4.0.0
```
title changes require a full version bump.

**Comment by tschmidtke-r7** [comment]
Changes applied, force pushed.

**Pull request closed without merging**